index.js

```diff
@@ -38,6 +38,10 @@ fs.readFile(userHomeDir + '/.boxrc.json', 'utf8', async function(err, data) {
     await connectToChannel(cmds[1], channel)
     process.exit(0)
   }
+  if (cmds.length === 1 && cmds[0] === 'me') {
```

Comment:
How is this going to work if the function is 'getMe' and you are checking for 
`...&& cmds[0] === 'me')`

Am I missing something?

Reply:
HAH!  I was washing my face and was like WTF.  You can call it whatever you want.